modules/oxide/src/less/theme/components/revisionhistory/revisionhistory.less

```diff
@@ -15,6 +15,7 @@
 @revisionhistory-text-lineheight: 24px;
 @revisionhistory-card-padding: @revisionhistory-base-value;
 @revisonhistory-box-shadow: none;
+@revsionhistory-card-label-color: #606973;
```

Comment:
```suggestion
@revisionhistory-card-label-color: #606973;
```
It is not being used anywhere

Reply:
We also shouldn't have fixed colours here, they should be theme variables.

Reply:
this is maybe not been used at all now. I'll check and delete it